apps/aecore/src/aec_block_key_candidate.erl

```diff
@@ -6,47 +6,28 @@
 %%%=============================================================================
 -module(aec_block_key_candidate).
 
--export([ create/2
-        , create_with_state/4
+-export([ create/1
         ]).
 
 -ifdef(TEST).
 -export([adjust_target/2]).
 -endif.
 
--export_type([block_info/0]).
-
 -include("blocks.hrl").
 
--type fees_info() :: #{txs := non_neg_integer(),
-                       gas := non_neg_integer()}.
--opaque block_info() :: #{trees := aec_trees:trees(),
-                          fees := fees_info(),
-                          txs_tree := aec_txs_trees:txs_tree(),
-                          adj_chain := [aec_headers:header()]}.
-
 %% -- API functions ----------------------------------------------------------
--spec create(aec_blocks:block() | aec_blocks:block_header_hash(), fees_info()) ->
-        {ok, aec_blocks:block(), block_info()} | {error, term()}.
-create(BlockHash, FeesInfo) when is_binary(BlockHash) ->
+-spec create(aec_blocks:block() | aec_blocks:block_header_hash()) ->
```

Comment:
This is legacy from pre-NG block creation it should be enough with the version that takes a hash... The other variant is only used in tests!??